lib/rust/ensogl/app/theme/hardcoded/src/lib.rs

```diff
@@ -650,6 +650,9 @@ define_themes! { [light:0, dark:1]
         slider {
             background = graph_editor::node::background , graph_editor::node::background;
             overshoot_limit = 60.0, 60.0;
+            text {
+                font = "mplus1p", "mplus1p";
+            }
```

Comment:
Indeed I should not have set that entry to an explicit font. I do need to set the text components' font to `"default"`, otherwise they use the old mono-spaced font.